### PR TITLE
[eFSR] Updating Error/Alert messages for debt selection

### PR DESCRIPTION
**Pull request by Dr-Pongo**
## Summary

- Moving Error and Alert cards to debt selection page (screenshots below)
- Fixing and updating skipped Cypress tests
- Adding new Cypress tests for new functionality. 

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#54005
- Fixed and un-skipping previously problematic tests
 #23917 
## Testing done

- Cypress tests passing locally
- Unit tests passing locally
- Manual testing also completed

## Screenshots

| Copay Error | Debt Error | Both Error | Both Zero |
| --- | --- | --- | --- |
| image | image | image | image | 

## What areas of the site does it impact?

*(Describe what parts of the site are impacted and*if*code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by OptionSelect — APPROVED** [review]
LGTM! 🚀

**Comment by amponce** [comment]
The code changes look great, the only issue I see is the stress test hangup

File: src/applications/financial-status-report/actions/copays.js [hunk 1]
```diff
@@ -42,11 +42,10 @@ export const getStatements = async dispatch => {
         statements: transformStatementData(data),
       });
     })
-    .catch(({ errors }) => {
-      const [error] = errors;
+    .catch(error => {
       Sentry.withScope(scope => {
         scope.setExtra('error', error);
-        Sentry.captureMessage(`medical_copays failed: ${error.detail}`);
+        Sentry.captureMessage(`medical_copays failed: ${error}`);
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/actions/copays.js:48]: Updating shortly to reduce information included

**Review by amponce — APPROVED** [review]
Love all these green checkmarks! ✅ 🚢

**Pull request merged**